Dockerfile

```
@@ -4,12 +4,12 @@ RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-
 ENV PATH="${PATH}:/root/.poetry/bin"
 WORKDIR /app
 COPY pyproject.toml poetry.lock /app/
-RUN poetry install
+RUN  poetry config virtualenvs.create false --local && poetry install
 COPY todo_app /app/todo_app/
 
 
 FROM base as production
-ENTRYPOINT [ "poetry", "run", "gunicorn", "-b", "0.0.0.0:5000", "todo_app.app:create_app()"]
+CMD poetry run gunicorn "todo_app.app:create_app()" --bind 0.0.0.0:$PORT
```

It's worth noting that in changing to use the PORT environment variable (which Heroku requires) we've broken the instructions in our Readme since we didn't tell users to set that variable. One nice solution is to set a default which matches our previous behaviour, and then to still allow Heroku to override that
We can do that with:
```suggestion
ENV PORT=5000
CMD poetry run gunicorn "todo_app.app:create_app()" --bind 0.0.0.0:$PORT
```